src/test/java/org/tugraz/sysds/test/TestUtils.java

```diff
@@ -1398,6 +1376,194 @@ public static void generateTestMatrixToFile(String file, int rows, int cols, dou
 		}
 	}
 
+	/**
+	 * <p>
+	 *     Generates a random FrameBlock with given parameters.
+	 * </p>
+	 */
+	public static FrameBlock generateRandomFrameBlock(int rows, int cols, ValueType[] schema, Random random){
+		String[] names = new String[cols];
+		for(int i = 0; i < cols; i++){
+			names[i] = schema[i].toString();
+		}
```

Comment:
Maybe this can become a one-liner? 
`Arrays.stream(schema).map(vt -> vt.name()).toArray(String[]::new)`